src/main/kotlin/racingcar/view/OutputView.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.view
+
+import racingcar.view.output.Output
+
+class OutputView(private val output: Output) {
+
+    fun show(movingDistances: List<Int>) {
+        val message = movingDistances.joinToString("") { "${this.statusBy(it)}\n" }
+        output.show(message)
+    }
+
+    fun statusBy(movingDistance: Int): String {
```

Comment:
`OutputView` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private fun statusBy(movingDistance: Int): String {
```